nixos/modules/services/desktop-managers/cosmic.nix

```diff
@@ -0,0 +1,157 @@
+# Copyright (c) 2024, Lily Foster <lily@lily.flowers>
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy of
+# this software and associated documentation files (the "Software"), to deal in
+# the Software without restriction, including without limitation the rights to
+# use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+# the Software, and to permit persons to whom the Software is furnished to do so,
+# subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+# FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+# COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+# IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+# CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+# This module was originally developed independently by lilystarlight in
+# https://github.com/lilyinstarlight/nixos-cosmic and was migrated to nixpkgs by Aleksana.
```

Comment:
I don't really think that this is necessary, as the entire nixpkgs is [already licensed under the MIT license](https://github.com/NixOS/nixpkgs/blob/master/COPYING).

Also, I would recommend to simply add lilystarlight as a Co-author in the commits, instead of specifying it here.

Reply:
@lilyinstarlight has indicated that they want the MIT license followed to the letter, and that includes attribution.